main.go

```diff
@@ -45,7 +48,6 @@ func main() {
 func httpInterceptor(router http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		startTime := time.Now()
-
```

Comment:
I had stuff there that i removed must have deleted that on accident without realizing